CMakeLists.txt

```diff
@@ -67,6 +67,11 @@ option(LARGEFILE             "Enable compiler definition -Duse_LARGEFILE"
 option(WITH_YAML             "Enable compiler definition -Duse_yaml"              OFF)
 option(USE_DEPRECATED_IO     "THIS OPTION HAS NO EFFECT AND WILL BE REMOVED IN A FUTURE RELEASE"   OFF)
 
+if(CMAKE_Fortran_COMPILER_ID MATCHES "NVHPC")
+# it seems nvidia does not support quad prec
+  set(ENABLE_QUAD_PRECISION OFF)
```

Comment:
Could you just add a small warning message just so its a little more transparent?